confluent_kafka/src/confluent_kafka.h

```diff
@@ -17,6 +17,7 @@
 #include <Python.h>
 #include <structmember.h>
 #include <pythread.h>
+#include <stdbool.h>
```

Comment:
Even though this is C99 there are some old shoddy toolchains out there, please use a standard `int` for bools to be safe.